### PR TITLE
NEXTEUROPA-5569 : Fix coding standards ec_* modules

### PR DESCRIPTION
**Pull request by senseca**

### CODE REVIEW

File: profiles/common/modules/features/ec_profiles/ec_profiles.install [hunk 1]
```diff
@@ -15,7 +15,7 @@ include_once 'ec_profiles.install.inc';
 function ec_profiles_enable() {
   drupal_set_message(t('EC Profiles feature is now active on your site.'));
```
Comment (gervasek) [inline profiles/common/modules/features/ec_profiles/ec_profiles.install]: There are 2 activation messages (only 1 is necessary):
``L16: drupal_set_message(t('EC Profiles feature is now active on your site.'));``
``L22: drupal_set_message(t('ec_profiles %v feature is now active on your site.', array('%v' => _ec_profiles_get_version())));``

**Comment by gervasek** [comment]
On  profiles/common/modules/features/ec_world_countries/ec_world_countries.install

1. ``L39  include_once 'includes/countries.php';``
Can be replaced by ``module_load_include('php', 'ec_world_countries', 'includes/countries');``

2. It's not necessary to translate the 'name' and 'description' of a taxonomy term.

```
L22  
$vocabulary = array(
      'name' => t('EC World Countries'),
      'machine_name' => 'ec_world_countries',
      'description' => t('A list of the world countries'),
      'hierarchy' => 1,
      'module' => 'ec_world_countries',
      'weight' => 1,
    );
L42
    $term = (object) array('vid' => $vid, 'name' => st($continent));
```

3. ``On ec_world_countries_enable`` : ``st()``, ``t()`` and ``get_t()`` are used to translate content.

**Comment by gervasek** [comment]
On ``nuts_regions.install.inc`` : Do not translate 
- taxonomy 'name' and 'description'
- field 'label'

**Comment by gervasek** [comment]
nuts_regions feature is a strange module:
- views are defined on the subfile ``views/nuts_regions.view`` and imported with the basic file ``nuts_regions.views_default.inc``
- ``hook_views_api`` is defined in ``myfeature.module`` instead of ``myfeature.feature.inc``

**Comment by senseca** [comment]
Changes done.
Concerning the nuts_regions feature, The subdirectory could be useful if we have couple of views. I don't know if it will be the case. In doubt, I let it like that.

**Comment by pfrenssen** [comment]
There is a merge conflict now.

File: profiles/common/modules/features/ec_content_slider/css/ec_content_slider.css [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+  /*-ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=70)"; *//* Mozilla Firefox */
+  /*filter: alpha(opacity=70)!important;*/ /* Internet Explorer */
+  background: rgba(0,0,0,0.7);/*chrome, mozilla*/
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorStr='#99000000', EndColorStr='#99000000'); /* Internet Explorer */
+  bottom: 0;
+  color: white;
+  left: 0;
+  padding: 5px;
+  position: absolute;
+  right: 0;
```
Comment (pfrenssen) [inline profiles/common/modules/features/ec_content_slider/css/ec_content_slider.css]: We can remove the commented out code.

File: profiles/common/modules/features/ec_embedded_video/ec_embedded_video.module [hunk 3]
```diff
@@ -78,10 +79,14 @@ function _ec_embedded_video_preprocess_media_vimeo_video(&$variables) {
...
-  // Don't use cookie complient for avportal video
+  $query = array(
+    'ref' => $variables['video_id'],
+    'starttime' => 0,
+    'endtime' => 0,
+  );
+  $variables['url'] = url('//ec.europa.eu/avservices/play.cfm', array('query' => $query, 'external' => TRUE));
+
+  // Don't use cookie complient for avportal video.
   $variables['ec_embedded_video_url'] = $variables['url'];
```
Comment (pfrenssen) [inline profiles/common/modules/features/ec_embedded_video/ec_embedded_video.module]: Minor spelling issue: 'complient' -> 'compliant'

File: profiles/common/modules/features/ec_embedded_video/ec_embedded_video.module [hunk 4]
```diff
@@ -49,7 +49,8 @@ function ec_embedded_video_theme_registry_alter(&$theme_registry) {
 
   // Add Cookie Consent support themeing and preprocess for supported filetypes.
```
Comment (pfrenssen) [inline profiles/common/modules/features/ec_embedded_video/ec_embedded_video.module]: Minor spelling issue: 'themeing' -> 'theming'.

File: profiles/common/modules/features/ec_embedded_image/ec_embedded_image.js [hunk 5]
```diff
@@ -1,10 +1,15 @@
+/**
+ * @file
+ * Javascript fil for embedded images (flickr).
+ */
```
Comment (pfrenssen) [inline profiles/common/modules/features/ec_embedded_image/ec_embedded_image.js]: Typo: missing 'e' after 'fil' :)

**Pull request merged**